### PR TITLE
[Wave 8 - ECard Transactions] Update Money Request View Fields to support Ecard transactions

### PR DESCRIPTION
**Pull request by grgia**
### Details

- Add Pending Field
- Add Card Menu Item
- Add support for original amount 
- Move Merchant up based on https://expensify.slack.com/archives/C027726KHT4/p1696943015610349?thread_ts=1696941226.715029&cid=C027726KHT4

Screenshot 2023-10-10 at 6 51 45 PM

### Fixed Issues

$ https://github.com/Expensify/App/issues/28835

### Tests

- [ ] Verify that no errors appear in the JS console
**We don't currently have card transactions in app, so I tested with mock data.**
- [ ] Create a normal IOU request. Open the transaction thread view for that request. Get the transactionID (I did this by logging it) 
- [ ] Use the following mock data to test cases (in console):

**PENDING, with Original Currency**
```js
Onyx.merge('cardList',
{
  "4": {
    "bank": "Expensify Card",
    "lastFourPan": "1000",
  }
});

Onyx.merge('transactions_3439799194391522645', {amount: 1000, currency: 'USD', cardID: "4",  modifiedMerchant: null, merchant: "United Airlines", mccGroup: "Goods", created: "2023-07-24 13:46:20", status:"Pending", transactionID: "3439799194391522645", originalAmount: 1000, originalCurrency: "GBP" });

```
image

**PENDING, with no Original Currency**
```js
Onyx.merge('cardList',
{
  4: {
    "bank": "Expensify Card",
    "lastFourPAN": "1000",
  }
});

Onyx.merge('transactions_3439799194391522645', {amount: 1000, currency: 'USD', cardID: 4,  modifiedMerchant: null, merchant: "United Airlines", mccGroup: "Goods", created: "2023-07-24 13:46:20", status:"Pending", transactionID: "3439799194391522645", originalAmount: 0, originalCurrency: '' });

```
image

**PENDING, with Original Currency**
```js
Onyx.merge('cardList',
{
  4: {
    "bank": "Expensify Card",
    "lastFourPAN": "1000",
  }
});

Onyx.merge('transactions_3439799194391522645', {amount: 1000, currency: 'USD', cardID: 4,  modifiedMerchant: null, merchant: "United Airlines", mccGroup: "Goods", created: "2023-07-24 13:46:20", status:"Posted", transactionID: "3439799194391522645", originalAmount: 1000, originalCurrency: "GBP" });

```
image

**PENDING, with no Original Currency**
```js
Onyx.merge('cardList',
{
  4: {
    "bank": "Expensify Card",
    "lastFourPAN": "1000",
  }
});

Onyx.merge('transactions_3439799194391522645', {amount: 1000, currency: 'USD', cardID: 4,  modifiedMerchant: null, merchant: "United Airlines", mccGroup: "Goods", created: "2023-07-24 13:46:20", status:"Posted", transactionID: "3439799194391522645", originalAmount: 0, originalCurrency: '' });

```
image

I also went through and verified that the border bottom works for all cases of scanning / posted transactions by manually setting to true/false:
**NONE**
Screenshot 2023-10-10 at 6 52 17 PM
**SCANNING**
Screenshot 2023-10-10 at 6 53 00 PM
**PENDING**
image
**BOTH**
image

### QA Steps

- [ ] Verify that no errors appear in the JS console

- [ ] Create any kind of manual request, verify that 'Merchant' field is displayed under description.

NO QA for card transactions changes, as we have not started sending card transactions data to App.

### PR Author Checklist

- [X] I wrote clear testing steps that cover the changes made in this PR
    - [X] I added steps for local testing in the `Tests` section
    - [X] I added steps for the expected offline behavior in the `Offline steps` section
    - [X] I added steps for Staging and/or Production testing in the `QA steps` section
    - [X] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [X] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [X] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [X] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [X] I ran the tests on **all platforms** & verified they passed on:
    - [X] Android: Native
    - [X] Android: mWeb Chrome
    - [X] iOS: Native
    - [X] iOS: mWeb Safari
    - [X] MacOS: Chrome / Safari
    - [X] MacOS: Desktop
- [X] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [X] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [X] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [X] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [X] I verified that comments were added to code that is not self explanatory
    - [X] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [X] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [X] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [X] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [X] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [X] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [X] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [X] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [X] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [X] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [X] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [X] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [X] I verified that if a function's arguments changed that all usages have also been updated correctly
- [X] If a new component is created I verified that:
    - [X] A similar component doesn't exist in the codebase
    - [X] All props are defined accurately and each prop has a `/** comment above it */`
    - [X] The file is named correctly
    - [X] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [X] The only data being stored in the state is data necessary for rendering and nothing else
    - [X] If we are not using the full Onyx data that we loaded, I've added the proper selector in order to ensure the component only re-renders when the data it is using changes
    - [X] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [X] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [X] All JSX used for rendering exists in the render method
    - [X] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [X] If any new file was added I verified that:
    - [X] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [X] If a new CSS style is added I verified that:
    - [X] A similar style doesn't already exist
    - [X] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [X] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [X] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [X] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [X] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [X] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [X] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [X] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Android: Native</summary>

</details>

<details>
<summary>Android: mWeb Chrome</summary>

</details>

<details>
<summary>iOS: Native</summary>

</details>

<details>
<summary>iOS: mWeb Safari</summary>

</details>

<details>
<summary>MacOS: Chrome / Safari</summary>

</details>

<details>
<summary>MacOS: Desktop</summary>

</details>

### CODE REVIEW

**Comment by grgia** [comment]
@ArekChr are you able to prioritize this? We have an external commit of Oct 22 for this project :)

**Comment by ArekChr** [comment]
Hi @grgia, yes, I will do review today

File: src/libs/CardUtils.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ function getMonthFromExpirationDateString(expirationDateString: string) {
  * @param cardID
  * @returns boolean
  */
-function isExpensifyCard(cardID: string) {
+function isExpensifyCard(cardID: number) {
```
Comment (ArekChr) [inline src/libs/CardUtils.ts:32]: I noticed that the `getCardDescription` function expects a cardID of type string. Should we also check and adjust the typing on line 44 to number?

File: src/libs/TransactionUtils.ts [hunk 2]
```diff
@@ -216,6 +217,21 @@ function getCurrency(transaction: Transaction): string {
...
 
+/**
+ * Return the original currency field from the transaction.
+ */
+function getOriginalCurrency(transaction: Transaction): string {
+    return transaction?.originalCurrency ?? '';
+}
+
+/**
+ * Return the original amount field from the transaction.
```
Comment (ArekChr) [inline src/libs/TransactionUtils.ts:228]:
```suggestion
 * Return the absolute value of the original amount field from the transaction.
```

File: src/libs/TransactionUtils.ts [hunk 3]
```diff
@@ -277,6 +300,27 @@ function isDistanceRequest(transaction: Transaction): boolean {
     return type === CONST.TRANSACTION.TYPE.CUSTOM_UNIT && customUnitName === CONST.CUSTOM_UNITS.NAME_DISTANCE;
 }
 
+function isExpensifyCardTransaction(transaction: Transaction): boolean {
```
Comment (ArekChr) [inline src/libs/TransactionUtils.ts:303]:
```suggestion
/**
 * Determine whether a transaction is made with an Expensify card.
 */
function isExpensifyCardTransaction(transaction: Transaction): boolean {
```

File: src/libs/TransactionUtils.ts [hunk 4]
```diff
@@ -277,6 +300,27 @@ function isDistanceRequest(transaction: Transaction): boolean {
...
+    if (!transaction.cardID) {
+        return false;
+    }
+    return isExpensifyCard(transaction.cardID);
+}
+
+function isPending(transaction: Transaction): boolean {
```
Comment (ArekChr) [inline src/libs/TransactionUtils.ts:310]:
```suggestion
/**
 * Check if a transaction status is set to PENDING.
 */
function isPending(transaction: Transaction): boolean {
```

File: src/libs/TransactionUtils.ts [hunk 5]
```diff
@@ -277,6 +300,27 @@ function isDistanceRequest(transaction: Transaction): boolean {
...
+    if (!transaction.status) {
+        return false;
+    }
+    return transaction.status === CONST.TRANSACTION.STATUS.PENDING;
+}
+
+function isPosted(transaction: Transaction): boolean {
```
Comment (ArekChr) [inline src/libs/TransactionUtils.ts:317]:
```suggestion
/**
 * Check if a transaction status is set to POSTED.
 */
function isPosted(transaction: Transaction): boolean {
```

**Review by ArekChr — COMMENTED** [review]
Overall the code looks good to me. I’ve noted a few minor points for consideration in the inline comments.

**Comment by ArekChr** [comment]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [ ] I verified the correct issue is linked in the `### Fixed Issues` section above
- [ ] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [ ] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [ ] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [ ] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [ ] I verified tests pass on **all platforms** & I tested again on:
    - [ ] Android: Native
    - [ ] Android: mWeb Chrome
    - [ ] iOS: Native
    - [ ] iOS: mWeb Safari
    - [ ] MacOS: Chrome / Safari
    - [ ] MacOS: Desktop
- [ ] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [ ] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [ ] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [ ] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [ ] I verified that comments were added to code that is not self explanatory
    - [ ] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [ ] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [ ] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [ ] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [ ] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [ ] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [ ] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [ ] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [ ] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [ ] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [ ] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [ ] If a new component is created I verified that:
    - [ ] A similar component doesn't exist in the codebase
    - [ ] All props are defined accurately and each prop has a `/** comment above it */`
    - [ ] The file is named correctly
    - [ ] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [ ] The only data being stored in the state is data necessary for rendering and nothing else
    - [ ] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [ ] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [ ] All JSX used for rendering exists in the render method
    - [ ] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [ ] If any new file was added I verified that:
    - [ ] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [ ] If a new CSS style is added I verified that:
    - [ ] A similar style doesn't already exist
    - [ ] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [ ] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [ ] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [ ] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [ ] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [ ] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [ ] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [ ] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>



</details>

<details>
<summary>Mobile Web - Chrome</summary>



</details>

<details>
<summary>Mobile Web - Safari</summary>



</details>

<details>
<summary>Desktop</summary>



</details>

<details>
<summary>iOS</summary>



</details>

<details>
<summary>Android</summary>



</details>

**Comment by grgia** [comment]
@ArekChr thank you for the review, will update tomorrow morning!

**Comment by grgia** [comment]
@ArekChr, pushed changes!

**Comment by ArekChr** [comment]
@grgia After simulating data, card data is not filled correctly. Could you provide more specific details on where you mocked Onyx?

Zrzut ekranu 2023-10-13 o 13 14 19

**Comment by grgia** [comment]
@ArekChr I fixed the data, it should be `lastFourPAN` capitalized. I also added protection against this in another PR here -https://github.com/Expensify/App/pull/29474/files#diff-19159f02504c955e4ad7285cc48ac7bfcc0e1cfbec74b371af5cb877311cc9b8

**Comment by ArekChr** [comment]
Great! I forgot to pull the latest changes, but after pulling I noticed now card is not visible

Zrzut ekranu 2023-10-13 o 13 26 37

**Comment by ArekChr** [comment]
This is for mock: 

```
        Onyx.merge('cardList', {
            4: {
                bank: 'Expensify Card',
                lastFourPan: 1000,
            },
        });

        Onyx.merge('transactions_5103348747052183509', {
            amount: 1000,
            currency: 'USD',
            cardID: '4',
            modifiedMerchant: null,
            merchant: 'United Airlines',
            mccGroup: 'Goods',
            created: '2023-07-24 13:46:20',
            status: 'Pending',
            transactionID: '5103348747052183509',
            originalAmount: 1000,
            originalCurrency: 'GBP',
        });
```

**Comment by grgia** [comment]
@ArekChr cardID should be number now, so:

```

       Onyx.merge('cardList', {
            4: {
                bank: 'Expensify Card',
                lastFourPan: 1000,
            },
        });

        Onyx.merge('transactions_5103348747052183509', {
            amount: 1000,
            currency: 'USD',
            cardID: 4,
            modifiedMerchant: null,
            merchant: 'United Airlines',
            mccGroup: 'Goods',
            created: '2023-07-24 13:46:20',
            status: 'Pending',
            transactionID: '5103348747052183509',
            originalAmount: 1000,
            originalCurrency: 'GBP',
        });
```

**Comment by ArekChr** [comment]
I changed to the type of number, but it still doesn't help. The card section is not visible; weird 🧐


```
        Onyx.merge('cardList', {
            4: {
                bank: 'Expensify Card',
                lastFourPan: 1000,
            },
        });

        Onyx.merge('transactions_5103348747052183509', {
            amount: 1000,
            currency: 'USD',
            cardID: 4,
            modifiedMerchant: null,
            merchant: 'United Airlines',
            mccGroup: 'Goods',
            created: '2023-07-24 13:46:20',
            status: 'Pending',
            transactionID: '5103348747052183509',
            originalAmount: 1000,
            originalCurrency: 'GBP',
        });
```

**Comment by ArekChr** [comment]
Ok i found kinda blocker, after logging the transaction it seems the cardID doesn't change at all

Zrzut ekranu 2023-10-13 o 13 48 49

**Comment by grgia** [comment]
> Ok i found kinda blocker, after logging the transaction it seems the cardID doesn't change at all

@ArekChr why do you have a transaction with that cardID? Curious how you set it before since you have to manual set cardID for testing in general

**Comment by ArekChr** [comment]
@grgia How do you mock data? I think this is essential

**Comment by grgia** [comment]
@ArekChr I use the Onyx merges

**Comment by ArekChr** [comment]
@grgia Yes, I see, but in which place?

**Comment by grgia** [comment]
@ArekChr in the console

**Comment by grgia** [comment]
@ArekChr here's an example of how I use merges to test in action:

https://github.com/Expensify/App/assets/38015950/b0848ed1-b37d-45fe-9970-3ea41e1d9588

**Review by ArekChr — APPROVED** [review]

**Comment by ArekChr** [comment]
All good, updated checklist 👌

**Comment by grgia** [comment]
@marcochavezf please let me know if you're not able to prioritize this

**Comment by marcochavezf** [comment]
I will review it today

File: src/components/MoneyRequestHeaderStatusBar.js [hunk 6]
```diff
@@ -1,24 +1,35 @@
...
+
+    /** Whether we show the border bottom */
+    shouldShowBorderBottom: PropTypes.bool.isRequired,
+};
+
+function MoneyRequestHeaderStatusBar({title, description, shouldShowBorderBottom}) {
+    const borderBottomStyle = shouldShowBorderBottom ? styles.borderBottom : {};
     return (
-        <View style={[styles.dFlex, styles.flexRow, styles.alignItemsCenter, styles.overflowHidden, styles.ph5, styles.pb3, styles.borderBottom, styles.w100]}>
+        <View style={[styles.dFlex, styles.flexRow, styles.alignItemsCenter, styles.flexGrow1, styles.overflowHidden, styles.ph5, styles.pb3, borderBottomStyle]}>
```
Comment (marcochavezf) [inline src/components/MoneyRequestHeaderStatusBar.js:21]: I noticed there are no screenshots for mobile platforms, and I think we'd need to ensure there are no visual regressions introduced in this PR

**Review by marcochavezf — APPROVED** [review]
The code LGTM

File: src/components/MoneyRequestHeaderStatusBar.js [hunk 7]
```diff
@@ -1,24 +1,35 @@
...
+    title: PropTypes.string.isRequired,
 
+    /** Banner Description */
+    description: PropTypes.string.isRequired,
+
+    /** Whether we show the border bottom */
+    shouldShowBorderBottom: PropTypes.bool.isRequired,
+};
+
+function MoneyRequestHeaderStatusBar({title, description, shouldShowBorderBottom}) {
```
Comment (youssef-lr) [inline src/components/MoneyRequestHeaderStatusBar.js:18]: Heads up this broke the status bar in the split details page and I fixed it here https://github.com/Expensify/App/pull/29568/commits/fc1e3ff36f52a2cea32a66ab788b1d7a19a489f6

**Pull request merged**